tests/test_hbg_dataset.py

```diff
@@ -22,29 +24,29 @@ def ontology():
                              ])
 
 
-@pytest.mark.parametrize('ontology, NWB_file', [(None, '2018_03_20_0005.nwb')], indirect=True)
-def test_main_abf(ontology, NWB_file):
+# @pytest.mark.parametrize('ontology, NWB_file', [(None, '2018_03_20_0005.nwb')], indirect=True)
```

Comment:
should probably remove commented out lines

Reply:
removed